src/guide/7-submodels/1-defining-a-submodel.rst

```diff
@@ -1,6 +1,6 @@
 Defining a Submodel
 ===================
-A submodel acts as a full model nested inside a parent model, the only requirement is that the model contains atleast one exit condition. Agent variables can be mapped between the two models, with each model also having private agent variables (or even private agents). Environment properties can also be shared with submodels, with the ability to mark them as constant within the submodel.
+A submodel acts as a full model nested inside a parent model, the only requirement is that the model contains atleast one exit condition. Agent variables can be mapped between the two models, with each model also having private agent variables (or even private agents). The agent ID of bound agents will always be mapped. Unbound agents which are recreated in each submodel run are not guaranteed to have the same ID between runs. Environment properties can also be shared with submodels, with the ability to mark them as constant within the submodel.
 
 Where an agent variable is mapped between model and submodel, the parent model's default value will always be used when creating new agents.
```

Comment:
> the parent model's default value will always be used
.. for any unmapped variables when creating new agents in the submodel.

Perhaps also note that any agents within the submodel, of unmapped types. Will disappear between calls to the submodel, and need to be recreated.